Kickstarter-iOS/Features/SetYourPassword/Controller/SetYourPasswordViewController.swift

```diff
@@ -130,14 +151,23 @@ public final class SetYourPasswordViewController: UIViewController {
 
     self.rootStackView.setCustomSpacing(Styles.grid(7), after: self.contextLabel)
     self.rootStackView.setCustomSpacing(Styles.grid(3), after: self.confirmPasswordTextField)
+
+    _ = (self.loadingIndicator, self.view)
+      |> ksr_addSubviewToParent()
```

Comment:
I haven't check the UI for this yet on device, but instead of of adding the `loadingIndicator` to the top level view, can we add it to the save button instead, kind of like the way we do "New payment method" in User Settings -> Payment Methods?

Reply:
So essentially replacing the save button with the loading indicator? Or having the indicator be inside the button and replace the "save" text?

Reply:
ah yea good catch. I can use `enableTextFieldsAndSaveButton()` to handle this. I'll update the failure case to reenable/show the button again as well.